e2e/common/traits/health_test.go

```diff
@@ -409,14 +406,15 @@ func TestHealthTrait(t *testing.T) {
 						return false
 					}
 
-					return data["check.kind"].(string) == "READINESS" && data["route.status"].(string) == "Stopped" && data["route.id"].(string) == "never-ready"
+					return r.Status == v1.HealthCheckStatusDown && data["check.kind"].(string) == "READINESS"
 				}))
 		})
 
 		t.Run("Startup condition with never ready route", func(t *testing.T) {
 			name := RandomizedSuffixName("startup-probe-never-ready-route")
 
-			g.Expect(KamelRunWithID(t, ctx, operatorID, ns, "files/NeverReady.java", "--name", name, "-t", "health.startup-probe-enabled=true", "-t", "health.startup-timeout=60").Execute()).To(Succeed())
+			g.Expect(KamelRunWithID(t, ctx, operatorID, ns, "files/NeverReady.java", "--name", name,
+				"-t", "health.enabled=true", "-t", "health.startup-probe-enabled=true", "-t", "health.startup-timeout=60").Execute()).To(Succeed())
```

Comment:
I think `health.enabled` is superfluous if any probe configuration is provided.

Reply:
It is, but it was explicitly checked (already before this pr) afterwards... but since I remove the check part I will remove this as well.